financialaid/views_test.py

```diff
@@ -252,25 +256,52 @@ def test_review_financial_aid_view_with_invalid_filter_and_sorting(self):
         url_with_bad_filter_and_bad_sorting = "{url}?sort_by=-askjdf".format(url=url_with_bad_filter)
         self.assert_http_status(self.client.get, url_with_bad_filter_and_bad_sorting, status.HTTP_200_OK)
 
+    def test_review_financial_aid_view_with_search(self):
+        """
+        Tests that ReviewFinancialAidView returns the expected results with search
+        """
+        FinancialAidFactory.create(tier_program=self.tier_programs["0k"], status=FinancialAidStatus.AUTO_APPROVED)
+        FinancialAidFactory.create(tier_program=self.tier_programs["0k"], status=FinancialAidStatus.APPROVED)
+        self.client.force_login(self.staff_user_profile.user)
+        # Works with search and filter
+        search_query = self.financialaid_approved.user.profile.first_name
+        search_url = "{path}?search_query={search_query}".format(
+            path=self.review_url_with_filter,
+            search_query=search_query
+        )
+        resp = self.assert_http_status(self.client.get, search_url, status.HTTP_200_OK)
+        resp_obj_id_list = resp.context_data["financial_aid_objects"].values_list("id", flat=True)
+        expected_obj_id_list = FinancialAid.objects.filter(
+            Q(user__profile__first_name__icontains=search_query) | Q(user__profile__last_name__icontains=search_query),
+            tier_program__program_id=self.program.id,
+            status=FinancialAidStatus.AUTO_APPROVED
+        ).order_by("user__profile__first_name").values_list("id", flat=True)  # Default sort field
```

Comment:
The default sort is `last_name`.

By the way, one of the problems with `UserFactory` is that it doesn't generate a profile using `ProfileFactory`, a blank profile is created automatically with a `post_save` signal. I think this works only because all profiles here are blank. Can you change instances of `FinancialAidFactory` to use profiles generated using `ProfileFactory`, maybe with a helper function?